Lagom 1.6.1

dwijnand opened the pull request.

dwijnand committed:
Lagom 1.6.1

ignasi35 commented on src/main/g8/build.sbt:
```diff
@@ -6,6 +6,7 @@ scalaVersion in ThisBuild := "2.13.0"
 
 val macwire = "com.softwaremill.macwire" %% "macros" % "2.3.3" % "provided"
 val scalaTest = "org.scalatest" %% "scalatest" % "3.0.8" % Test
+val scalaXml = "org.scala-lang.modules" %% "scala-xml" % "1.2.0" % Test
```
Wait WAT? Why?

dwijnand replied:
Trying random things to fix it.  But it isn't a scala-xml thing here.

ignasi35 commented on project/build.properties:
```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.3.8
```
Want to bump `1.3.8` in `lagom/lagom-java.g8` too?

dwijnand replied:
No, feel free.

ignasi35 replied:
https://github.com/lagom/lagom-java.g8/pull/77

dwijnand committed:
Upgrade to ScalaTest 3.1.1

Something through Lagom 1.6.1 uses ScalaTest 3.1, which is binary incompatbile with 3.0, so we need to upgrade.
